atox/src/main/res/values/strings.xml

```diff
@@ -122,20 +122,20 @@
     <string name="pref_heading_address">Address</string>
     <string name="pref_heading_port">Port</string>
     <string name="pref_heading_proxy">Proxy</string>
-    <string name="warn_proxy_broken">Bad proxy configured. Fix or disable before using app.</string>
+    <string name="warn_proxy_broken">Set up your proxy correctly (or turn it off) before using the app.</string>
```

Comment:
Can you remove the full stop now that this is a single sentence?

Reply:
If you want, we can talk about how the text is set up in the app, but that might be better suited to IRC or Tox than GitHub comments? :P Right now, the only text that has any full stops at the end are the ones that are multiple sentences.